index.js

```diff
@@ -510,6 +511,9 @@ class GithubScm extends Scm {
             // Fetch a pull request
             command.push(`echo Fetching PR and merging with ${branch}`);
             command.push(`$SD_GIT_WRAPPER "git fetch origin ${prRef}"`);
+
+            command.push(`export PR_BRANCH_NAME=origin/${config.prBranchName}`);
```

Comment:
what about PR from forked repo? it doesn't have to be from `origin` right?

Reply:
@DekusDenial review thank you 👍 
When I tried on the forked repo, I was able to get the `prBranchName`.
Certainly it doesn't have to be the origin, is there anything else?
I have no good idea,
e.g.
fork/prBranchName?

Reply:
@ibu1224 if we stick to the terminology used by github, use `upstream` for the base repo and `origin` for your forked repo. If the merging branch on the pr is from the same base repo you will use `origin` for both base and merging branches. I think this is pretty common in git as well.

Reply:
OK.
I will fix it to check fork and add `upstream`.